frontend/src/pages/journal/components/journal-sidebar/journal-sidebar.tsx

```diff
@@ -1,17 +1,22 @@
 import {
+  Button,
   Card,
+  Link,
   Sidebar,
   SidebarBody,
   SidebarHeader,
 } from '#libs/components/components.js';
+import { AppRoute } from '#libs/enums/app-route.enum.js';
```

Comment:
```suggestion
import { AppRoute } from '#libs/enums/enums.js';
```

Reply:
But you left a comment in the file that is located in the frontend, even though I haven't touched the frontend at all

Reply:
I figured later, cause you had a lot of files from Frontend and backend in your commits. You can just change it now, so we wouldn't forget to fix it later.

Reply:
I didn't change anything in the frontend folder
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/0bc7e99d-c712-46a3-ac0d-fd4c737875c5)